frontends/ol-components/src/components/Button/Button.stories.tsx

```diff
@@ -42,7 +42,7 @@ const meta: Meta<typeof Button> = {
       control: { type: "select" },
     },
     edge: {
-      options: ["circular", "rounded", "none"],
+      options: ["circular", "rounded"],
```

Comment:
edge="none" is not in figma; it was added for resource cards, but that is handled a little differently now (changed in #1180 )